### DISCUSSION TITLE
Many to Many relationship field type

### DISCUSSION THREAD

**Discussion by tom-sk** [statamic-rad-pack/runway#286] [Feature Requests]
Hi @duncanmcclean,

Thanks for creating this package, it has already been really useful!

I was just wondering if you have plans to include a Many to Many relationship field in the future? 

Or should I just try and modify the field types that are already there?

**Comment by duncanmcclean** [comment]
Have you tried using a HasMany relationship fieldtype on both sides of the relationship?

**Reply by tom-sk** [reply]
I'll give that a go but I thought with that method only one side could have more than one?

I thought to do this I needed a pivot table to manage the relationship?

**Reply by tom-sk** [reply]
I have used the hasMany on both sides and it has worked, but now the sort_order is in the pivot table and isn't being updated?

**Reply by duncanmcclean** [reply]
Please open an issue regarding the sort order issue.
